hedera-node/hedera-app/src/itest/java/signatures/SignatureVerificationTest.java

```diff
@@ -70,6 +70,8 @@
 class SignatureVerificationTest implements Scenarios {
 
     private static final long DEFAULT_CONFIG_VERSION = 1L;
+    private static final VersionedConfiguration CONFIGURATION =
```

Comment:
We should really have a helper class for that...

Reply:
Do you mean a builder for `VersionedConfiguration`?